### PR TITLE
Fix: Added check for removing Stderr File for Sucessful Submission(#…

### LINKED ISSUE #1175: Remove Stderr file for the successful submissions
## Observed behavior

Currently when there is a successful submission, then also a `stderr_file` is created for that submission which leads to the creation of a blank file that is of no use and adds a small overhead in processing the successful submissions.

## Expected behavior

When there is a successful submission, then `stderr_file` field should be set None instead of creating a blank file.

### PR DESCRIPTION
**Pull request by Ram81**
Fixes #1175.
@deshraj Please Review. 
temp

### CODE REVIEW

File: scripts/workers/submission_worker.py [hunk 1]
```diff
@@ -398,9 +398,10 @@ def run_submission(challenge_id, challenge_phase, submission_id, submission, use
     with open(stdout_file, 'r') as stdout:
         stdout_content = stdout.read()
         submission.stdout_file.save('stdout.txt', ContentFile(stdout_content))
-    with open(stderr_file, 'r') as stderr:
-        stderr_content = stderr.read()
-        submission.stderr_file.save('stderr.txt', ContentFile(stderr_content))
+    if (submission_status is not Submission.FINISHED):
```
Comment (deshraj) [inline scripts/workers/submission_worker.py]: I think the check should be 

`if (submission_status is not Submission.FAILED):`

Please correct me if I am wrong.

Comment (Ram81) [reply scripts/workers/submission_worker.py]: @deshraj if the submission is failed then you want an Stderr file to be generated right?

Comment (Ram81) [reply scripts/workers/submission_worker.py]: @deshraj you want an Stderr file to be generated in case of submission Failure, right?

Comment (deshraj) [reply scripts/workers/submission_worker.py]: Yeah.

Comment (Ram81) [reply scripts/workers/submission_worker.py]: And @deshraj 
if (submission_status is not Submission.FAILED):
this check will save an stderr file for every successful submission, right!

**Comment by Ram81** [comment]
@deshraj I updated the check condition, it is less ambiguous now. Please review

**Comment by deshraj** [comment]
Looks good. Merging it.

**Comment by deshraj** [comment]
Thanks @Ram81 for fixing this. :)

**Comment by Ram81** [comment]
@deshraj can you review #1276, It's pending from few days :smile:

**Pull request merged**